### DISCUSSION TITLE
Noob tentando resolver um programa simples

### DISCUSSION THREAD

**Discussion by LucianeFernandesRoque** [frontendbr/forum#1836] [Perguntas e Respostas]
### Galera.
Sou formada em sistemas para internet, estou voltando aos estudos na área da programação;

vamos lá, criei o código que ele sorteia um numero de 0 a 10,até ai blz, tudo normal,criei uma variável tentativas, só que ao invés do programa dar mais uma chance ao usuário, ele mostra 3 x a msg vc errou.
O código esta em txt tbm, desde já agradeço.
[bazinga.txt](https://github.com/frontendbr/forum/files/4810305/bazinga.txt)

######

`meta charset="UTF-8">
<input/> 
<button>Compare com o meu segredo</button>
<scrip
				function pulaLinha() {

				document.write("<br>");	
				}

				function mostra(frase) {

				document.write(frase);
				pulaLinha();
				}
				
			
		        alert("Qual numero você acha que eu tenho no meu segredo?");

				function verifica(){ // inicio da chave verifica
				var tentativas = 1;

   				for (tentativas = 1;  tentativas <= 3;  tentativas++) {
   				var segredo = Math.round(Math.random() * 10);
                var input = document.querySelector("input");
                input.focus();
                input.value = " ";

				if (input.value == segredo){
				mostra("Você acertou");
				break;

                }if (input.value != segredo) {
				alert("você errou");
				alert("tente novamente");
				}

				

			}
		

		    mostra("Suas tentativas acabaram.");
	}

				// fecha chave verifica
            var button = document.querySelector("button");
            button.onclick = verifica;
`</script>``

`

**Comment by highlabs** [comment]
Nesse caso creio que o melhor seria usar um `return` em vez de `break`. Mais aqui: https://pt.stackoverflow.com/a/24985

Mas nesse caso, você não precisa de um loop. Somente uma função para verificar cada vez que o botão é clicado, incrementando a variável "tentativas".

Exemplo: https://gist.github.com/highlabs/9a037355398ebe66464b754fead530bd
Ou no codepen: https://codepen.io/Dobrado/pen/OJMgMNL
